source/configure/configuration-settings.rst

```diff
@@ -4592,6 +4592,10 @@ This setting isn't available in the System Console and can only be set in ``conf
 | This feature's ``config.json`` setting is ``"EnableLocalMode": false`` with options ``true`` and ``false``. |
 +-------------------------------------------------------------------------------------------------------------+
 
+.. tip::
+
+  If you're seeing the error ``socket file "/var/tmp/mattermost_local.socket" doesn't exists, please check the server configuration for local mode``, this error can be resolved by setting this configuration setting to ``true``.
```

Comment:
nit: this error will come from `mmctl` so, maybe we can start the sentence with something like:
> When trying to use local mode with mmctl, having an error like ...

Reply:
Great suggestion! Thank you!